Sources/CentralManager/CentralManager.swift

```diff
@@ -241,6 +241,10 @@ extension CentralManager.DelegateWrapper: CBCentralManagerDelegate {
             } catch {
                 Self.logger.error("Received onDidConnect without a continuation!")
             }
+            
+            self.context.eventSubject.send(
+                .didConnectPeripheral(peripheral: Peripheral(peripheral))
```

Comment:
Cool! This will help track auto-connecting devices. It seems `didConnectPeripheral` is not defined in this PR. Are you missing a commit?

Reply:
That's exactly the case I was looking to solve here :) ... I'll check again see if I missed it.

Reply:
It's committed. Just in another file 
https://github.com/manolofdez/AsyncBluetooth/pull/10/commits/0979f8fd26de3627cac33a8cc28e7313dd39e859#diff-a6691c4c155669f5dab4b68e5c3891cce2ed556041e56a70638a6037ae04c0bd

Reply:
Nice! I must have missed it.